.teamcity/_self/projects/WPComTests.kt

```diff
@@ -128,3 +129,58 @@ private object Gutenberg : BuildType({
 		}
 	}
 })
+
+private object VisualRegressionTests : BuildType({
+	name = "Visual Regression Tests"
+	description = "Runs visual regression tests"
+
+	artifactRules = """
+		reports => reports
+	""".trimIndent()
+
+	vcs {
+		root(Settings.WpCalypso)
+		cleanCheckout = true
+	}
+
+	steps {
+		bashNodeScript {
+			name = "Prepare environment"
+			scriptContent = """
+				export NODE_ENV="test"
+
+				# Install modules
+				${_self.yarn_install_cmd}
+			"""
+			dockerImage = "%docker_image_e2e%"
```

Comment:
You should be able to remove the lines specifying the `dockerImage` in each step (theoretically!) The project settings have it defined to the e2e image, so that _should_ be default for every step in the WPComTests project already.

Reply:
I took it out, will add back later if it doesn't work!

Reply:
you can also remove it from the other script steps in the file :)